src/components/DocumentationTopic/DocumentationHero.vue

```diff
@@ -0,0 +1,126 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2021 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <div class="documentation-hero" :style="styles">
+    <NavigatorLeafIcon :type="type" key="first" class="background-icon first-icon" with-colors />
+    <NavigatorLeafIcon :type="type" key="second" class="background-icon second-icon" with-colors />
+    <div class="documentation-hero__content">
+      <slot />
+    </div>
+  </div>
+</template>
+
+<script>
+
+import NavigatorLeafIcon from 'docc-render/components/Navigator/NavigatorLeafIcon.vue';
+import { TopicTypeColorsMap, TopicTypeAliases, TopicTypeColors } from 'docc-render/constants/TopicTypes';
+
+export default {
+  name: 'DocumentationHero',
+  components: { NavigatorLeafIcon },
+  props: {
+    type: {
+      type: String,
+      required: true,
+    },
+  },
+  computed: {
+    // get the alias, if any, and fallback to the `teal` color
+    color: ({ type }) => TopicTypeColorsMap[TopicTypeAliases[type] || type] || TopicTypeColors.teal,
+    styles: ({ color }) => ({
+      // use the color or fallback to the gray secondary, if not defined.
+      '--accent-color': `var(--color-type-icon-${color}, var(--color-figure-gray-secondary))`,
+    }),
+  },
+};
+</script>
+
+<style scoped lang='scss'>
+@import 'docc-render/styles/_core.scss';
+
+$doc-hero-gradient-background: dark-color(fill-tertiary) !default;
+$doc-hero-overlay-background: transparent !default;
+$doc-hero-icon-opacity: 1 !default;
+$doc-hero-icon-color: dark-color(fill-secondary) !default;
+
+.documentation-hero {
+  background: dark-color(fill);
+  color: light-color(fill);
+  overflow: hidden;
+  text-align: center;
+  padding-top: rem(40px);
+  padding-bottom: 40px;
+  position: relative;
+
+  // gradient
+  &:before {
+    content: '';
+    background: $doc-hero-gradient-background;
+    position: absolute;
+    width: 100%;
+    left: 0;
+    top: -50%;
+    height: 150%;
+    right: 0;
+  }
+
+  // black overlay
+  &:after {
+    background: $doc-hero-overlay-background;
+    opacity: 0.4;
+    width: 100%;
+    position: absolute;
+    content: '';
+    height: 100%;
+    left: 0;
+    top: 0;
+  }
+
+  .background-icon {
+    color: $doc-hero-icon-color;
+    position: absolute;
+    display: block;
+    width: 250px;
+    height: 250px;
+    opacity: $doc-hero-icon-opacity;
+
+    /deep/ svg {
+      width: 100%;
+      height: 100%;
+    }
+
+    &.first-icon {
+      left: 0;
+      top: 100%;
```

Comment:
This is probably more of a design thing - but with 100%, a lot of the icon is cut off, so you can't actually see which letter the icon is. I experimented with a few values and 80% looks pretty good IMO.
<img width="841" alt="Screen Shot 2022-02-16 at 11 10 27 PM" src="https://user-images.githubusercontent.com/87735557/154423881-b6e481af-4ef4-4a63-a388-fe162670c669.png">

Reply:
Will check with Design.